CRM/Utils/Check/Component/Env.php

```diff
@@ -963,14 +963,25 @@ public function checkPHPIntlExists() {
   public function checkWkHtmlToPDFPath() {
     $messages = [];
     $wkhtmltopdfPath = CRM_Core_Config::singleton()->wkhtmltopdfPath;
-    if (!empty($wkhtmltopdfPath) && !file_exists($wkhtmltopdfPath)) {
-      $messages[] = new CRM_Utils_Check_Message(
+    if (!empty($wkhtmltopdfPath) {
+      // check and ensure that this leads to the wkhtmltopdf binary
+      // and it is a valid executable binary
+      // Only check the first space separated piece to allow for a value
+      // such as /usr/bin/xvfb-run -- wkhtmltopdf (CRM-13292)
+      $pieces = explode(' ', $fields['wkhtmltopdfPath']);
+      $path = $pieces[0];
+      if (
+        !file_exists($path) ||
+        !is_executable($path)
+      ) {  
+        (
```

Comment:
```suggestion
      if (!file_exists($path) || !is_executable($path)) {  
        $messages[] = new CRM_Utils_Check_Message(
```

Reply:
I find inline conditionals more readable (unless they are huge). And the variable assignment + class constructor got accidentally omitted from your patch.

Reply:
Also the indentation below this comment is incorrect now, needs to be indented +2 spaces.
Thanks for the PR!

Reply:
Thanks for the verification, I was doing the editing in github's web UI editor which leaves some things to be desired. Do you need me to make those changes?

Reply:
Yes please if you can.

Reply:
Done. https://github.com/civicrm/civicrm-core/pull/19843/commits/d0a3a25137832df1c57389051fb90daea2b5a840